package.json

```
@@ -1,14 +1,31 @@
 {
-  "name": "safe-event-emitter",
+  "name": "@metamask/safe-event-emitter",
   "version": "1.0.1",
-  "description": "",
+  "description": "An EventEmitter that isolates the emitter from errors in handlers",
   "main": "index.js",
+  "publishConfig": {
+    "registry": "https://registry.npmjs.org/",
+    "access": "public"
+  },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "tsc --project .",
+    "test": "jest",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
```

Wouldn't this also lint the generated `.js` files? That doesn't seem ideal

---

Oh, they're ignored in the `.eslintignore` file. Might as well remove the extension here as well.

---

There are _some files that are linted, i.e. the ESLint config itself and the Jest config

---

Ah I see - I missed that.

Probably not any `.jsx` files though?
```suggestion
    "lint": "eslint . --ext .js,.ts,.tsx"
```

---

Updated